oteapi/strategies/parse/image.py

```diff
@@ -51,26 +68,59 @@ class ImageParserResourceConfig(ResourceConfig):
 class SupportedFormat(Enum):
     """Supported formats for `ImageDataParseStrategy`."""
 
-    JPEG = "JPEG"
-    JPEG2000 = "JPEG2000"
-    PNG = "PNG"
-    GIF = "GIF"
-    TIFF = "TIFF"
-    EPS = "EPS"
+    jpeg = "JPEG"
+    jpg = "JPEG"
+    jp2 = "JPEG2000"
+    png = "PNG"
+    gif = "GIF"
+    tiff = "TIFF"
+    tif = "TIFF"
+    eps = "EPS"
 
 
 class SessionUpdateImageParse(SessionUpdate):
-    """Configuration model for ImageParse."""
+    """Configuration model for ImageParse.
 
-    content: bytes = Field(..., description="Parsed output from ImageParse.")
-    format: SupportedFormat = Field(..., description="The format of the parsed image.")
-    cropped: bool = Field(..., description="Whether or not the image has been cropped.")
+    See [Pillow handbook](https://pillow.readthedocs.io/en/stable/handbook/concepts.html) for more details
+    on `image_mode`, `image_palette`, and `image_info`.
+    """
+
+    image_key: str = Field(
+        ...,
+        description="Key with which the image content is stored in the data cache.",
+    )
+    image_size: Tuple[int, int] = Field(
+        ...,
+        description="Image size (width, height).",
+    )
+    image_mode: str = Field(
+        ...,
+        description="Image mode. Examples: 'L', 'P', 'RGB', 'RGBA'...",
+    )
+    image_palette_key: str = Field(
+        None,
```

Comment:
The type should either be `Optional[str]` or the default value should be a string - can be an empty string (`""`).

Reply:
ok